tests/model_inheritance/tests.py

```diff
@@ -158,6 +160,43 @@ def test_create_diamond_mti_default_pk(self):
         with self.assertNumQueries(4):
             common_child.save()
 
+    def test_create_diamond_mti_common_parents(self):
+        with self.assertNumQueries(4):
+            itrmp = ItalianRestaurantManyParents.objects.create(
+                name="Ristorante Miron",
+                address="1234 W. Ash",
+            )
+
+        italian_resturant_place = itrmp.italianrestaurant_ptr.place_ptr
+        itrmp_place = itrmp.place_ptr_two
+        italian_resturant_resturant = itrmp.italianrestaurant_ptr.restaurant_ptr
+        itrmp_resturant = itrmp.restaurant_ptr
+
+        # Some draft tests to test regression
+        self.assertEqual(italian_resturant_place, itrmp_place)
+        self.assertEqual(italian_resturant_resturant, itrmp_resturant)
+        self.assertEqual(itrmp_resturant.place_ptr, itrmp_place)
+        self.assertEqual(itrmp.name, "Ristorante Miron")
+        self.assertEqual(itrmp.address, "1234 W. Ash")
+
+    def test_check_on_diamond_mti_common_parents(self):
+        class ItalianRestaurantManyParentsAbstract(ItalianRestaurant, Place):
+            pass
+
+        clash = Restaurant._meta.get_field("place_ptr")
+        field = ItalianRestaurantManyParentsAbstract._meta.parents.get(Place)
```

Comment:
```suggestion
        field = ItalianRestaurant._meta.get_field("place_ptr")
```

Reply:
Will update this. Let me know what you think about the previous [comment](https://github.com/django/django/pull/16952#discussion_r1233776158)

Reply:
Updated suggestion.